doc/configuration.rst

```diff
@@ -659,10 +659,18 @@ directive so that you can easily add a reduced version of the Gallery to
 your Sphinx documentation ``.rst`` files. The mini-gallery directive therefore
 supports passing a list (space separated) of any of the following:
 
-* full qualified name of object (see :ref:`references_to_examples`)
+* full qualified name of object (see :ref:`references_to_examples`) - this
+  adds all examples where the object was used in the code or referenced in
+  the example text
 * pathlike strings to example Python files, including glob-style
   (see :ref:`file_based_minigalleries`)
 
+To use object names, you must enable backreference generation, see
+:ref:`references_to_examples` for details.
+If backreference generation is not enabled, object entries to the
+:class:`~sphinx_gallery.directives.MiniGallery` directive will be ignored
+and all entries will be treated as pathlike strings.
+
 .. _references_to_examples:
 
 Add mini-galleries for API documentation
```

Comment:
@larsoner reading:

> The former is useful for auto-documenting functions/methods/attributes/objects
that are used and classes that are explicitly instantiated. The generated links
are called implicit backreferences.

> The latter is useful for classes that are
typically implicitly returned rather than explicitly instantiated (e.g.,
:class:`matplotlib.axes.Axes` which is most often instantiated only indirectly
within function calls). Such links are called explicit backreferences.

From the explanation it sounds like code use backreferences should be 'explicit' and text backreferences should be called 'implicit'? This explanation is a bit confusing at the very least.
Do you know the orginal naming details?

Reply:
Not 100% sure what I meant :) But I think the idea was maybe that the things that are explicit in the code (function calls, instantiations) are implicitly/automatically added as backrefs by SG. Things that are only implicit in the code (like `Axes`) might need to be made explicit.

Reply:
Cool, I will try to improve the docs. I want to keep the names as this is what they are called in the code.